portfolio/src/main/webapp/index.html

```diff
@@ -4,12 +4,45 @@
     <meta charset="UTF-8">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="font-awesome-4.7.0/css/font-awesome.min.css">
```

Comment:
Instead of having a copy of this stylesheet, you should load it from the CDN. CDNs will provide much better cacheability.

Reply:
I think I might just remove that stylesheet altogether